system/Filters/DebugToolbar.php

```diff
@@ -57,6 +57,7 @@ class DebugToolbar implements FilterInterface
 	 */
 	public function before(RequestInterface $request)
 	{
+		config('toolbar')->enabled = true;
```

Comment:
Setting it here makes the setting pointless since it will always be set when you get into the View class, doesn't it?